### PR TITLE
update code owners

### PR DESCRIPTION
**Pull request by xiangyan99**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -30,7 +30,7 @@
 /sdk/identity/                                                       @mccoyp @schaabs @xiangyan99
 
 # PRLabel: %Event Hubs
-/sdk/eventhub/                                                       @annatisch @yunhaoling @swathipil @rakshith91
+/sdk/eventhub/                                                       @annatisch @kashifkhan @swathipil @rakshith91
```
Comment (jsquire) [inline .github/CODEOWNERS:33]: Should we consider adding Libba to Event Hubs and Service Bus?

**Review by jsquire — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -102,7 +102,7 @@
 /sdk/deviceupdate/                                                   @dpokluda @sedols
 
 # PRLabel: %Event Grid
-/sdk/eventgrid/                                                      @lmazuel @yunhaoling @swathipil @rakshith91
+/sdk/eventgrid/                                                      @lmazuel @kashifkhan @swathipil @rakshith91
```
Comment (rakshith91) [inline .github/CODEOWNERS:105]:
```suggestion
/sdk/eventgrid/                                                      @lmazuel @rakshith91
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -132,7 +132,7 @@
 /sql/sql/                                                            @jaredmoo
 
 # PRLabel: %Service Bus
-/sdk/servicebus/                                                     @annatisch @yunhaoling @swathipil @rakshith91
+/sdk/servicebus/                                                     @annatisch @kashifkhan @swathipil @rakshith91
```
Comment (rakshith91) [inline .github/CODEOWNERS:135]:
```suggestion
/sdk/servicebus/                                                     @annatisch @kashifkhan @swathipil @l0lawrence
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (xiangyan99) [reply .github/CODEOWNERS]: Added.

**Review by rakshith91 — APPROVED** [review]

**Pull request merged**